modules/tx_index_schema.sql

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
What is the use of this file `tx_index_schema.sql`?

Reply:
This is a (very slightly) optimized sql schema for the Postgres trx store backend (an alternative to the leveldb backend). I was using it for doing some data collection for cofi, I wanted to keep it around.

Reply:
Ok. I am just curious how it is used, as I don't see it being referred to anywhere else.

Reply:
You need to manually apply it to the postgres database that you spin up for the tx_index backend. It isn't connected to anything in this repo, but is useful when spinning up nodes.

Reply:
I see. Might be good to include it inside a custom build output together with the executable so that it can be loaded from there.

Reply:
I don't think there is a specific build output that it is relevant for. You could theoretically use this schema with any cosmos SDK based node. Happy to move the file somewhere more relevant or rename it better or add a readme. Just don't want to lose this work.